sdk/cosmos/azure-cosmos-benchmark/pom.xml

```diff
@@ -477,6 +477,25 @@ Licensed under the MIT License.
                   </archive>
                 </configuration>
               </execution>
+              <execution>
+                <id>make-assembly-ctl</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>single</goal>
+                </goals>
+                <configuration>
+                  <descriptorRefs>
+                    <descriptorRef>jar-with-dependencies</descriptorRef>
+                  </descriptorRefs>
+                  <outputDirectory>${project.build.directory}/ctl</outputDirectory>
```

Comment:
Didn't we have an existing `jar-with-dependencies` plugin already ?
Why can't we use the existing one instead of creating another one here ?

Reply:
Yes we could use that , but it has version in it . And docker script need exact file name to run. Either i change the existing name to static without version in it . But in doing so devs who compare benchmark across different version , it would be difficult for them to maintain or they have to rename the jar each time.

Reply:
Can wildcard copy followed by move leveraged?

Reply:
We can use wild card while copy , but script while running need exact name

Reply:
I also agree with Kushagra's point, if we can we should avoid duplicating the build config for the uber jar.

@simplynaveen20 in your dockerfile,  you can have a command to discover the jar file name and then copy it to /usr/app/azure-cosmos-benchmark-jar-with-dependencies.jar 
isn't that possible?

Reply:
Done